subgraph.template.yaml

```diff
@@ -27,6 +27,47 @@ dataSources:
           handler: handleOwnerAddition
       file: ./src/luw-mapping.ts
 
+  - kind: ethereum/contract
+    name: StartrailRegistry_legacy
+    network: mainnet
+    source:
+      address: "0x7f556e211a3E4b57d005D3aa49a31306FA8Bb34d"
+      abi: StartrailRegistry
+      startBlock: 10720664 # contract creation block
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - SRR
+      abis:
+        - name: StartrailRegistry
+          file: ./abis/legacy/StartrailRegistry.json
+      file: ./src/srr-mapping.ts
+      eventHandlers:
+        - event: Transfer(indexed address,indexed address,indexed uint256)
```

Comment:
@noyonthe1 There is a lot of overlap between the legacy and non-legacy entries. What do you think about putting only legacy events in the legacy entry and the current entry has the full set of current events only? For each of the contracts the same pattern would apply.

In terms of ordering of contract entries in the template yaml. Would it make sense to put like contract entries together with legacy always coming second? That way as you scan down they are grouped together but recent entry comes first.

Reply:
done into this commit https://github.com/startbahn/startrail-subgraph/pull/48/commits/734c27ed5c2496e90df6ec9c72632bb04e2ebefe